keys/employees-keys/README.md

```diff
@@ -3,172 +3,117 @@
 This directory keeps 3mdeb employees keys and explains the procedure for adding
 new keys.
 
+## Table of contents
+
+* [Adding key to the repository](#adding-key-to-the-repository)
+* [FAQ](#faq)
+  - [What to do when this key expires?](#what-to-do-when-this-key-expires)
+  - [If we extend expiry every time, why not set "key does not
+    expire"?](#if-we-extend-expiry-every-time-why-not-set-key-does-not-expire)
+  - [What to do when private key was lost or
```

Comment:
This link does not render correctly

Reply:
I have no idea why this doesn't work and previous works fine: cca3480e6135

Reply:
@pietrushnic @macpijan there is no such section as `What to do when private key was lost or compromised?` that is why link does not work.

Reply:
Ok, fixed 282ec0ab04ea

Reply:
@macpijan can we close this conversation?

Reply:
@macpijan ping?